include/config.h

```diff
@@ -48,6 +51,16 @@ namespace detail {
 		std::optional<bool> m_enable_device_profiling;
 		size_t m_graph_print_max_verts = 200;
 		int m_dry_run_nodes = 0;
+
+		// required for test
+		std::vector<size_t> m_devices_list;
+
+		// parsing functions
+		size_t parse_validate_graph_print_max_verts(const std::string_view str);
+		bool parse_validate_profile_kernel(const std::string_view str);
+		size_t parse_validate_dry_run_nodes(const std::string_view str);
+		std::vector<size_t> parse_validate_devices(const std::string_view str);
+		int parse_validate_forge_wg(const std::string_view str);
```

Comment:
I frist tried to implement it as `void` and came across that error. Hence I made it `int` because it does not get used. But I see your point. I will change it to `bool`